lib/Signer.js

```diff
@@ -93,6 +93,13 @@ function Signer(newSigner = {}) {
      * @type {Date}
      */
     embeddedSigningUrl: undefined,
+      
+     /**
+     * Forces document to be emailed in the case embedded signing is enabled for this template
+     * Usually starts with 1 for the first Signer.
+     * @type {Boolean}
+     */
+    setDeliverEmail: undefined,
```

Comment:
I think it has to be:

`deliverEmail: undefined, `

as the field in the JSON request is then called `deliver_email`